app/assets/stylesheets/skins/pghkids/contexts/_pghkids.scss

```diff
@@ -5,9 +5,9 @@
 }
 
 .site-header {
-  background: url(/assets/skins/pghkids/header_mobile.png) no-repeat $header-bg-color;
+  background: url(/production/assets/skins/pghkids/header_mobile.png) no-repeat $header-bg-color;
```

Comment:
I don't think we need the full path here. We can, I think, go with something more like this: https://github.com/pgharts/trusty-festivity-extension/blob/master/app/assets/stylesheets/first-night-stylesheets/contexts/_first-night.scss#L4

Though in this case it would be, I *think*, `url(skins/pghkids/header_mobile.pgh)`. That way the asset pipeline can figure out the total url based on whatever environment it's in.